.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,70 @@
+---
+name: Release
+about: Cut a Crossplane release
+labels: release
+---
+
+<!--
+Issue title should be in the following format:
+
+    Cut vX.Y.Z Release
+-->
+
+<!--
+Please assign the release owner to the issue.
+-->
+
+### Release Information
+
+**Type:**
+<!-- Select One -->
+- [ ] Minor
+- [ ] Patch
+
+**Version:** <!-- e.g. v1.3.0 -->
+
+**Release Date:** <!-- e.g. Jun 29, 2021 -->
+
+### Steps
+
+<!-- Please complete the following steps in order. -->
+
+I have:
+<!-- Please uncomment the following section only if cutting a minor release. These should be completed at the beginning of [Code Freeze](https://crossplane.io/docs/v1.2/reference/release-cycle.html#code-freeze). -->
```

Comment:
Should we wrap this? I'm not sure whether GitHub wraps this when you use the template.

Reply:
We could wrap this one -- though it won't actually ever be uncommented. Some of the placholder links did not render well when wrapped, so this template is somewhat pseudo-wrapped right now.

Reply:
My auto-wrapper doesn't wrap comments I have now discovered, but everything that is not a comment or may be un-commented has now been wrapped :+1: